CONTRIBUTING.rst

```diff
@@ -91,43 +95,32 @@ Using a custom ``virtualenv``
   Unfortunately using ``setup.py develop`` is not possible with this
   project, because it uses `namespace packages`_.
 
-Using ``tox``
+Using ``nox``
 =============
 
-- To test your changes, run unit tests with ``tox``::
-
-    $ tox -e py27
-    $ tox -e py34
-    $ ...
-
-- If you'd like to poke around your code in an interpreter, let
-  ``tox`` install the environment of your choice::
-
-    $ # Install only; without running tests
-    $ tox -e ${ENV} --recreate --notest
+We use `nox`_ to instrument our tests.
```

Comment:
This is invalid, not sure why:

![screenshot from 2017-08-08 09-57-07](https://user-images.githubusercontent.com/520669/29084150-f6fc3be2-7c1f-11e7-8789-7edb17b2814e.png)